### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by dongzitai**

### CODE REVIEW

File: tests/swoole_process/exception.phpt [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Process6 extends AbstractProcess
 --EXPECTF--
 Fatal error: Uncaught Error: Call to undefined function AAAA() in %s:%d
 Stack trace:
-#0 %s(%d): Process6->run()
+#0 %s(%d): Process6->run()vi
```
Comment (matyhtf) [inline tests/swoole_process/exception.phpt:48]: 多了vi字符串？

**Comment by dongzitai** [comment]
对 另一个修复的test是因为8.1没通过



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/25/2022 11:38 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [swoole/swoole-src] Fix tests (PR #4892) |

@matyhtf commented on this pull request.

In tests/swoole_process/exception.phpt:

> @@ -45,7 +45,7 @@ class Process6 extends AbstractProcess

 --EXPECTF--

 Fatal error: Uncaught Error: Call to undefined function AAAA() in %s:%d

 Stack trace:

-#0 %s(%d): Process6->run()

+#0 %s(%d): Process6->run()vi



多了vi字符串？

**Comment by dongzitai** [comment]
我再检查一下



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/25/2022 11:38 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [swoole/swoole-src] Fix tests (PR #4892) |

@matyhtf commented on this pull request.

In tests/swoole_process/exception.phpt:

> @@ -45,7 +45,7 @@ class Process6 extends AbstractProcess

 --EXPECTF--

 Fatal error: Uncaught Error: Call to undefined function AAAA() in %s:%d

 Stack trace:

-#0 %s(%d): Process6->run()

+#0 %s(%d): Process6->run()vi



多了vi字符串？

**Comment by matyhtf** [comment]
> 对 另一个修复的test是因为8.1没通过
> […](#)
> ---- Replied Message ---- | From | ***@***.***> | | Date | 10/25/2022 11:38 | | To | ***@***.***> | | Cc | ***@***.******@***.***> | | Subject | Re: [swoole/swoole-src] Fix tests (PR #4892) | @matyhtf commented on this pull request. In tests/swoole_process/exception.phpt:
> @@ -45,7 +45,7 @@ class Process6 extends AbstractProcess
> --EXPECTF-- Fatal error: Uncaught Error: Call to undefined function AAAA() in %s:%d Stack trace: -#0 %s(%d): Process6->run() +#0 %s(%d): Process6->run()vi 多了vi字符串？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

对，这个需要修复

File: tests/swoole_process/exception.phpt (hunk 1 above)
Comment (dongzitai) [reply tests/swoole_process/exception.phpt]: 本次提交确实存在问题，已撤回

**Pull request merged**